specification/adhybridhealthservice/resource-manager/Microsoft.ADHybridHealthService/stable/2014-01-01/ADHybridHealthService.json

```diff
@@ -0,0 +1,5641 @@
+      "get": {
+        "tags": [ "Adds" ],
+        "description": "Gets the service configurations.",
+        "x-ms-examples": {
+          "configuration_listAddsConfigurations": {
+            "$ref": "./examples/AddsConfiguration.json"
+          }
+        },
+        "operationId": "configuration_listAddsConfigurations",
+        "parameters": [
+          {
+            "name": "serviceName",
+            "in": "path",
+            "description": "The name of the service.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "grouping",
+            "in": "query",
+            "description": "The grouping for configurations.",
+            "required": false,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": " ",
+            "schema": {
+              "$ref": "#/definitions/AddsConfiguration"
+            }
+          },
+          "400": {
+            "description": "Bad Request. This is the response code when the tenant is not onboarded to Azure Active Directory Connect Health.Exception will not be thrown."
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/providers/Microsoft.ADHybridHealthService/addsservices/{serviceName}/dimensions/{dimension}": {
+      "get": {
+        "tags": [ "Adds" ],
+        "description": "Gets the dimensions for a given dimension type in a server.",
+        "x-ms-examples": {
+          "dimensions_listAddsDimensions": {
+            "$ref": "./examples/Dimensions.json"
+          }
+        },
+        "operationId": "dimensions_listAddsDimensions",
+        "parameters": [
+          {
+            "name": "serviceName",
+            "in": "path",
+            "description": "The name of the service.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "dimension",
+            "in": "path",
+            "description": "The dimension type.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": " ",
+            "schema": {
+              "$ref": "#/definitions/Dimensions"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/providers/Microsoft.ADHybridHealthService/addsservices/{serviceName}/addsservicemembers": {
+      "get": {
+        "tags": [ "Adds" ],
+        "description": "Gets the details of the Active Directory Domain servers, for a given Active Directory Domain Service, that are onboarded to Azure Active Directory Connect Health.",
+        "x-ms-examples": {
+          "addsServicemembers_list": {

```

Comment:
@gamitra - Last capitalization correction => "addsServiceMembers_foo"

Reply:
Yeah, looks like this was missed. I will send an update right away